src/extensions/maui/Microsoft.DotNet.UpgradeAssistant.Extensions.Maui/ExtensionManifest.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+  "ExtensionName": "MAUI",
+
+  "TryConvert": {
```

Comment:
There's already a setting for try-convert options ([TryConvertProjectConverter](https://github.com/dotnet/upgrade-assistant/blob/main/src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Extensions.Default/ExtensionManifest.json#L10)). The only reason it doesn't work for your purposes is that we're not registering it as an extension option (it's [added as a normal IOptions instead](https://github.com/dotnet/upgrade-assistant/blob/main/src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Extensions.Default/DefaultExtensionServiceProvider.cs#L38)).

If possible, it would be nice to re-use the existing config section and just register it as an extension option instead.

Reply:
I'd be fine reconciling the two into one. However, it will make it so that the path to try-convert will be configurable by an extension (maybe we want that?)

Reply:
Good point. I think that's actually an improvement, though.

Reply:
I made the updates to the PR except for this change. Should I combine the options? or leave it as is for the moment?

Reply:
let's combine it